isaaclab_arena/environments/arena_env_builder.py

```diff
@@ -27,11 +27,12 @@
 class ArenaEnvBuilder:
     """Compose IsaacLab Arena → IsaacLab configs"""
 
-    DEFAULT_SCENE_CFG = InteractiveSceneCfg(num_envs=4096, env_spacing=30.0, replicate_physics=False)
-
     def __init__(self, arena_env: IsaacLabArenaEnvironment, args: argparse.Namespace):
         self.arena_env = arena_env
         self.args = args
+        self.DEFAULT_SCENE_CFG = InteractiveSceneCfg(
```

Comment:
I'm wondering why this is all caps. I'm thinking let's change the name `DEFAULT_SCENE_CFG` -> `interactive_scene_cfg`. At this point it's not really the default any more. It can be modified, and then is directly used.

Thanks for putting this as a class member to get people more flexability.